src/drive/web/modules/drive/Toolbar/components/DocumentQualification.jsx

```diff
@@ -81,6 +82,15 @@ class DocumentQualification extends Component {
       }
     }
   }
+  /**
+   * Method used to synchronize our internal state and
+   * our parent state if needed
```

Comment:
Nit: "Synchronizes internal state and parentState if needed" is shorter and
has the same information ;)